### PR TITLE
adds information on BAMM CLI

### PR DESCRIPTION
**Pull request by eriksven**
Adds information BAMM CLI to Readme files and updates links to SDS documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,18 +25,16 @@
...
 
 The SDS SDK contains artifacts and resources for all parties that intent to use, extend or integrate with the BAMM
-Aspect Meta Model, e.g. Solution Developers, Domain Experts or OEMs.
+Aspect Meta Model, e.g., Solution Developers, Domain Experts or OEMs.
 
-At its core are components, which help working with the BAMM Aspect Meta Model (BAMM). The SDK comprises components to
-load and validate models such as the one depicted below and generate artifacts such as DTOs, static classes,
-documentation and so on.
-![BAMM Aspect Meta Model (BAMM) Elements](documentation/developer-guide/modules/getting-started-guide/images/solution-with-movement-aspect.svg)
+At its core are components, which help to work with the BAMM Aspect Meta Model (BAMM). The SDK comprises components to
```
Comment (atextor) [inline README.md:30]: Remove comma

File: README.md [hunk 2]
```diff
@@ -25,18 +25,16 @@
...
+load and validate models generate artifacts such as static classes, documentation, DTO (data transfer object) and so on.
```
Comment (atextor) [inline README.md:31]: "DTOs (data transfer objects)"? As "static classes" is in plural, DTOs should maybe also be plural

**Comment by eriksven** [comment]
The idea for this PR is based on the discussion in https://github.com/OpenManufacturingPlatform/sds-sdk/issues/51 . 

However, this PR does not close the Issue as there we additional question in the issue regarding the build.

File: README.md [hunk 3]
```diff
@@ -25,18 +25,16 @@
...
 
 To fully leverage BAMM, provided components include language-specific meta model implementations, code generators,
 validators etc.
 
-This document provides an overall overview of the SDK and the concepts applied throughout it. Detailed documentation and
-concepts for each component can be found in the respective subfolders.
+This document provides an overall overview of the SDK, and the concepts applied throughout it. Detailed documentation and
+concepts for each component can be found in the respective subfolder.
```
Comment (atextor) [inline README.md:37]: This refers to the subfolders in which the different modules reside, so this should probably stay plural.

File: tools/bamm-cli/README.md [hunk 4]
```diff
@@ -1,6 +1,15 @@
-# BAMM Aspect Meta Model Command Line Tool
+# BAMM Aspect Meta Model Command Line Interface
 
-## Building
+The BAMM CLI bundles the functionality provided in the SDS SDK in a CLI application to be used in you terminal for ad-hoc usage outside an IDE.
+For details on the usage and offered functionalities see [Running](#Running)
```
Comment (atextor) [inline tools/bamm-cli/README.md:4]: Add full stop at the end.

**Review by atextor — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (eriksven) [reply README.md:37]: My thought (and probably the one of the IDE highlighting the plural as an error) was that each component has its individual respective subfolder. At the same time, the subfolder may also contain subfolders, so I will change it back to plural.

**Comment by eriksven** [comment]
@atextor I updated the PR

**Review by atextor — APPROVED** [review]

**Review by atextor — APPROVED** [review]

**Pull request merged**